lib/util.js

```diff
@@ -357,9 +361,8 @@ module.exports = {
   },
   promisify,
   stripVTControlCharacters,
-  toUSVString(input) {
-    return StringPrototypeToWellFormed(`${input}`);
-  },
+  toUSVString: deprecate(toUSVString, 'util.toUSVString() is deprecated. ' +
+    'Please use String.prototype.toWellFormed() instead.', 'DEP0175'),
```

Comment:
```suggestion
    'Use String.prototype.toWellFormed() instead.', 'DEP0175'),
```

Reply:
Why? The wording is taken from the deprecations.md file.